discover/portal/articles/210-setting-up/02-instance-settings/02-instance-settings.markdown

```diff
@@ -0,0 +1,217 @@
+functionality and don't have anything to do with the integration options on the
+other tabs. There are several general authentication settings:
+
+-   Authenticate by email address (default), screen name, or User ID (a
+    numerical ID auto-generated in the database--not recommended).
+
+-   Enable/Disable automatic log in. If enabled, a User can check
+    a box which will cause the site to "remember" the login information by
+    placing a cookie on the browser. If disabled, Users must always log in
+    manually.
+
+-   Enable/Disable forgotten password functionality.
+
+-   Enable/Disable request password reset links.
+
+-   Enable/Disable account creation by strangers. If running an Internet
+    site, leave this enabled so visitors can create accounts on your site.
+
+-   Enable/Disable account creation by those using an email address in the
+    domain of the company running the site (which is set on the General page of
+    Instance Settings). This is handy if you're using Liferay to host both
+    internal and external web sites. Make sure all internal IDs are created by
+    administrators but external Users can register for IDs themselves.
+
+-   Enable/Disable email address verification. If enabled,
+    Users receive a verification email with a link back to the virtual instance,
+    verifying that the email address they entered is valid.
+
+By default, all settings except for the last are enabled. User authentication by
+email address is an important default for the following reasons:
+
+1.  An email address is unique to the User who owns it.
+
+2.  People remember their email addresses. A Users who hasn't logged in for a
+    while could forget their screen name.
+
+3.  If email address isn't used to authenticate, a User might change her email
+    address then forget to update the email address in her profile. If this
+    occurs, no notifications sent by the virtual instance will reach the User.
+    Keeping the email address at the forefront of a User's mind when she logs in
+    helps ensure the User keeps it current. 
+
+To leanr to configure a third party authentication service or set up Single Sign
+On (SSO), see the 
+[security documentation](/discover/deployment/-/knowledge_base/7-1/liferay-portal-security).
+
+## Users [](id=Users)
+
+The *Users* section has three tabs: Fields, Reserved Credentials and Default
+User Associations.
+
+In the Fields tab, enable/disable these fields on the Add/Edit User Form:
+
+- Autogeneration of screen names
+
+- Birthday field
+
+- Gender field
+
+In the Reserved Credentials tab, specify the screen names and email addresses
+Users aren't allowed to register with. Use this feature to prevent Users from
+creating IDs that look like administrative IDs or that have reserved words in
+their names.
+
+Default User Associations has four fields for listing (one per line) Sites,
+Organization Sites, Soles, and User Groups you want new Users to become members
```

Comment:
Instead of saying "to become members of" which only applies to Sites and User Groups. I would use/add "to be assigned" or something like that.

Also, I don't think it's "Organization Sites" but rather "Organizations" but I may be wrong. The difference is in whether the user is really made part of the listed Organization(s), so that the org admin can manage the user account.

Reply:
It's Organization sites in my build. Here's the help text: 
>Enter the default organization site names per line that are associated with newly created users.

Reply:
It's weird. I wonder if the help text is also wrong.

Could you test it?

Reply:
Wow, I assumed you were correct. but after testing, no, this seems to assign new users as organization site members without assigning them to the organization.

It sounds crazy, but they can see the organization site in My Sites, while the Organizations section of their User form says This User Does not Belong to an Organization.

This is truly confusing.